android/src/main/java/com/clevertap/react/CleverTapEvent.kt

```diff
@@ -0,0 +1,35 @@
+package com.clevertap.react
+
+enum class CleverTapEvent(val eventName: String, val bufferable: Boolean = false) {
+
+    CLEVERTAP_PROFILE_DID_INITIALIZE("CleverTapProfileDidInitialize", bufferable = true),
+    CLEVERTAP_PROFILE_SYNC("CleverTapProfileSync"),
+    CLEVERTAP_IN_APP_NOTIFICATION_DISMISSED("CleverTapInAppNotificationDismissed", bufferable = true),
+    CLEVERTAP_IN_APP_NOTIFICATION_SHOWED("CleverTapInAppNotificationShowed", bufferable = true),
+    CLEVERTAP_INBOX_DID_INITIALIZE("CleverTapInboxDidInitialize", bufferable = true),
```

Comment:
I am not sure about the initialization of the Inbox. If you are certain that it will not be triggered before RN is loaded, I can remove it from the list. I added it just to be safe as currently making things bufferable is not expensive.

Reply:
Ideally the initialisation for inbox would be the done on a button click or after the component is loaded. But clients might integrate non-ideally and hence we can be safe by keeping it bufferable